content/docs/guides/flyway-multiple-environments.md

```diff
@@ -0,0 +1,211 @@
+
+    <CodeBlock shouldWrap>
+
+    ```bash
+    flyway.url=jdbc:postgresql://ep-cool-darkness-123456.us-east-2.aws.neon.tech:5432/neondb
+
+    flyway.user=alex
+
+    flyway.password=AbC123dEf
+
+    flyway.locations=filesystem:/home/alex/flyway-9.22.3/sql
+
+    flyway.baselineOnMigrate=true
+    ```
+
+    </CodeBlock>
+
+    - The `flyway.locations` setting tells Flyway where to look for your migration files. We'll create them in the `/sql` directory in a later step.
+    - The `flyway.baselineOnMigrate=true` setting tells Flyway to perform a baseline action when you run the `migrate` command on a non-empty schema with no Flyway schema history table. The schema will then be initialized with the `baselineVersion` before executing migrations. Only migrations above the `baselineVersion` will then be applied. This is useful for initial Flyway deployments on projects with an existing database. You can disable this setting by commenting it out again or setting it to false after applying your first migration on the database.
+
+## Create a migration
+
+Create a migration file called `V2__Add_people.sql`, add it to your Flyway `/sql` directory, and add the following statements to the file:
+
+```bash
+insert into person (ID, NAME) values (1, 'Alex');
+insert into person (ID, NAME) values (2, 'Mr. Lopez');
+insert into person (ID, NAME) values (3, 'Ms. Smith');
+```
+
+In the following steps, you'll run the migration on each environment in order by specifying the environment's configuration file in the `flyway migrate` command. You'll start with your `development` environment, then `staging`, and then finally, `production`.
+
+### Run the migration on your development environment
+
+```bash
+flyway migrate -configFiles="conf/env_dev.conf"
+```
+
+If the command was successful, you’ll see output similar to the following:
+
+```bash
+Database: jdbc:postgresql://ep-nameless-unit-49929920.us-east-2.aws.neon.tech/neondb (PostgreSQL 15.4)
+Schema history table "public"."flyway_schema_history" does not exist yet
+Successfully validated 1 migration (execution time 00:00.199s)
+Creating Schema History table "public"."flyway_schema_history" with baseline ...
+Successfully baselined schema with version: 1
+Current version of schema "public": 1
+Migrating schema "public" to version "2 - Add people"
+Successfully applied 1 migration to schema "public", now at version v2 (execution time 00:00.410s)
+A Flyway report has been generated here: /home/alex/flyway-9.22.3/report.html
+```
+
+### Run the migration on your staging environment
+
+```bash
+flyway migrate -configFiles="conf/env_staging.conf"
+```
+
+If the command was successful, you’ll see output similar to that shown above.
+
+### Run the migration on your product environment
+
```

Comment:
```suggestion
Switch to the Flyway root directory and run the following command:
```

Reply:
Because we were just in the `sql` directory the the configFile is a relative path, the command will fail if you try to run while still in the `sql` directory.